### PR TITLE
fix some typos

### PR DESCRIPTION
**Pull request by zsqw123**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -224,7 +224,7 @@ fun myFunction(dep: Dep) {
 }
 ```
 
-You can then use the type alias anywhere and you will be provided with a function that calls the top-level one with the
+You can then use the type alias anywhere, and you will be provided with a function that calls the top-level one with the
```
Comment (evant) [inline README.md:227]: I'm not sure the comma here is necessary?

File: README.md [hunk 2]
```diff
@@ -248,7 +248,7 @@ fun myFunction(dep: Dep, arg: String): String = ...
 
 ### Scopes
 
-By default kotlin-inject will create a new instance of a dependency each place it's injected. If you want to re-use an
+By default, kotlin-inject will create a new instance of a dependency each place it's injected. If you want to re-use an
```
Comment (evant) [inline README.md:251]: Same here

**Review by evant — COMMENTED** [review]
Good catches!

File: README.md (hunk 2 above)
Comment (zsqw123) [reply README.md:251]: ok, maybe I can revert it, I will only fix some typos :)

**Review by evant — APPROVED** [review]

**Pull request merged**